x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/connector/constants.ts

```diff
@@ -104,18 +116,24 @@ export const CONNECTORS_DICT: Record<string, ConnectorClientSideDefinition> = {
     externalDocsUrl: '',
     icon: CONNECTOR_ICONS.amazon_s3,
   },
+  salesforce: {
+    docsUrl: docLinks.connectorsSalesforce,
+    externalAuthDocsUrl: '',
+    externalDocsUrl: '',
```

Comment:
we ok missing docs for FF i assume?

Reply:
Yeah these auth docs don't show up if they're not filled, so that's fine.